lms/static/scripts/frontend_apps/config.js

```diff
@@ -16,6 +16,14 @@ import { createContext } from 'preact';
  * @prop {object} [data]
  */
 
+/**
+ * Configuration determining which "mode" the frontend app should operate in.
+ * Each mode maps to an app-level UI component.
+ *
+ * @typedef {'basic-lti-launch'|'content-item-selection'|'error-dialog'|'oauth2-redirect-error'} AppMode
```

Comment:
This didn't exist and it seemed like perhaps it should.

Reply:
Yes, this looks like a useful addition.